prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -53,6 +55,13 @@ class ReportFunction {
         val defaults: Map<String, String> = emptyMap(),
         val routeTo: List<String> = emptyList(),
         val report: Report? = null,
+        val verbose: String = "",
+    )
+
+    data class ItemJurisdictionRouting(
+        val reportIndex: Int,
+        val trackingId: String?,
+        val destinations: Map<Organization.Jurisdiction, List<String>> = emptyMap(),
```

Comment:
This could be changed back to a `List<String>` vs a `Map` keyed on Jurisdiction. Initially this map was for being able to find items that only routed to FEDERAL receivers. On the fence if we keep it this way or simplify it.

Reply:
Yes, per my other comment, I think you could get rid of the keying by Jurisdiction.